docs/source/reference/config.rst

```diff
@@ -191,10 +191,22 @@ Available fields and semantics:
 
     # Reserved capacity (optional).
     #
-    # The specific reservation to be considered when provisioning clusters on GCP.
-    # SkyPilot will automatically prioritize this reserved capacity (considered as
-    # zero cost) if the requested resources matches the reservation.
+    # Whether to prioritize reservations when provisioning clusters on GCP. This
+    # is useful when you have `automatically consumed reservation` in your GCP
+    # project and want to prioritize them over on-demand resources.
+    #
+    # Default: false.
+    prioritize_reservations: false
```

Comment:
Should we call this `auto_consumed_reservations`? Since we seem to use these 2 fields for 2 types of rsvp's, respectively.

Reply:
`auto_consumed_reservations` could be a bit confusing, as no matter we set this to `true` or not, it will always use the reservations. This option is to only make the optimizer to prioritize the region/zones with auto reservations.